packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -280,20 +282,42 @@ class AzureDevOpsServerManager {
     return {error: firstError, workItems: workItemReferences}
   }
 
-  async getUserStories(instanceId: string, queryString: string | null, isWIQL: boolean) {
-    if (isWIQL) {
-      const customQueryString = queryString
-        ? `Select [System.Id], [System.Title], [System.State] From WorkItems Where ${queryString}`
-        : "Select [System.Id], [System.Title], [System.State] From WorkItems Where [System.WorkItemType] = 'User Story' AND [State] <> 'Closed' AND [State] <> 'Removed' order by [Microsoft.VSTS.Common.Priority] asc, [System.CreatedDate] desc"
+  async getUserStories(
+    instanceId: string,
+    queryString: string | null,
+    projectKeyFilters: string[] | null,
+    isWIQL: boolean
+  ) {
+    let customQueryString = ''
```

Comment:
+1 as we're not using `customQueryString` in the first loop, it might easier to read if it's moved to line 304.5